### ISSUE TITLE
pyRevit 2.6.7 crashing Revit on loading of plugin

### ISSUE THREAD

**Issue by gnomesoup** [eirannejad/pyRevit#942]
**Describe the bug**
Revit crashes after installation of pyRevit 4.7.6. If I uninstall and reinstall using pyRevit 2.7.5, the plugin loads just fine. No pyRevit log window appears before crash.
[journal.0007.txt](https://github.com/eirannejad/pyRevit/files/4939084/journal.0007.txt)
[journal.0007.worker1.log](https://github.com/eirannejad/pyRevit/files/4939086/journal.0007.worker1.log)

**To Reproduce**
1. Install pyRevit using the pyRevit_4.7.6_signed.exe (.NET requirements already installed)
2. Open Revit 

**Expected behavior**
Revit should load the pyRevit plugin without crashing

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: WIndows 10
 - pyRevit 4.7.6
 - pyRevit Environment: 
```
==> Registered Clones (full git repos)
==> Registered Clones (deployed from archive/image)
master | Deploy: "basepublic" | Branch: "master" | Version: "4.7.6" | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master"
==> Attachments
master | Product: "2019.2.2 Hotfix" | Engine: 277 | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master" | Manifest: "C:\Users\mpfammatter\AppData\Roaming\Autodesk\Revit\Addins\2019\pyRevit.addin"
master | Product: "2018 First Customer Ship" | Engine: 277 | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master" | Manifest: "C:\Users\mpfammatter\AppData\Roaming\Autodesk\Revit\Addins\2018\pyRevit.addin"
master | Product: "2017.2.5 Security Fix" | Engine: 277 | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master" | Manifest: "C:\Users\mpfammatter\AppData\Roaming\Autodesk\Revit\Addins\2017\pyRevit.addin"
master | Product: "2016 Service Pack 2" | Engine: 277 | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master" | Manifest: "C:\Users\mpfammatter\AppData\Roaming\Autodesk\Revit\Addins\2016\pyRevit.addin"
master | Product: "2015 Update Release 3" | Engine: 277 | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master" | Manifest: "C:\Users\mpfammatter\AppData\Roaming\Autodesk\Revit\Addins\2015\pyRevit.addin"
==> Installed Extensions
pyWHA | Type: Unknown | Repo: "" | Installed: "W:\01 BIM\pyWHA\pyWHA.extension"
==> Default Extension Search Path
C:\Users\mpfammatter\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
W:\01 BIM\pyWHA
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
2019.2.2 Hotfix | Version: 19.2.20.24 | Build: 20190808_0900(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2019"
2018 First Customer Ship | Version: 18.0.0.420 | Build: 20170223_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2018"
2017.2.5 Security Fix | Version: 17.0.1169.0 | Build: 20190508_0315(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2017"
2016 Service Pack 2 | Version: 16.0.490.0 | Build: 20150714_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2016"
2015 Update Release 3 | Version: 15.0.207.0 | Build: 20140606_1530(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2015"
==> Running Revit Instances
==> User Environment
Microsoft Windows 10 [Version 10.0.18362]
Executing User: WRIGHT\mpfammatter
Active User:
Admin Access: No
%APPDATA%: "C:\Users\mpfammatter\AppData\Roaming"
Latest Installed .Net Framework: 4.8
No .Net Target Packs are installed.
No .Ne-Core Target Packs are installed.
pyRevit CLI 0.25.0.0
```

**Additional context**
journal file attached.

**Comment by gnomesoup** [comment]
**Describe the bug**
Revit crashes after installation of pyRevit 2.7.6. If I uninstall and reinstall using pyRevit 2.7.5, the plugin loads just fine. No pyRevit log window appears before crash.
[journal.0007.txt](https://github.com/eirannejad/pyRevit/files/4939084/journal.0007.txt)
[journal.0007.worker1.log](https://github.com/eirannejad/pyRevit/files/4939086/journal.0007.worker1.log)
[journal.0007.txt](https://github.com/eirannejad/pyRevit/files/4939092/journal.0007.txt)




**To Reproduce**
Steps to reproduce the behavior:
1. Install pyRevit using the pyRevit_2.7.6_signed.exe (.NET requirements already installed)
2. Open Revit 

**Expected behavior**
Revit should load the pyRevit plugin without crashing

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: WIndows 10
 - pyRevit 2.7.6
 - pyRevit Environment: 
```
==> Registered Clones (full git repos)
==> Registered Clones (deployed from archive/image)
master | Deploy: "basepublic" | Branch: "master" | Version: "4.7.6" | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master"
==> Attachments
master | Product: "2019.2.2 Hotfix" | Engine: 277 | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master" | Manifest: "C:\Users\mpfammatter\AppData\Roaming\Autodesk\Revit\Addins\2019\pyRevit.addin"
master | Product: "2018 First Customer Ship" | Engine: 277 | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master" | Manifest: "C:\Users\mpfammatter\AppData\Roaming\Autodesk\Revit\Addins\2018\pyRevit.addin"
master | Product: "2017.2.5 Security Fix" | Engine: 277 | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master" | Manifest: "C:\Users\mpfammatter\AppData\Roaming\Autodesk\Revit\Addins\2017\pyRevit.addin"
master | Product: "2016 Service Pack 2" | Engine: 277 | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master" | Manifest: "C:\Users\mpfammatter\AppData\Roaming\Autodesk\Revit\Addins\2016\pyRevit.addin"
master | Product: "2015 Update Release 3" | Engine: 277 | Path: "C:\Users\mpfammatter\AppData\Roaming\pyRevit-Master" | Manifest: "C:\Users\mpfammatter\AppData\Roaming\Autodesk\Revit\Addins\2015\pyRevit.addin"
==> Installed Extensions
pyWHA | Type: Unknown | Repo: "" | Installed: "W:\01 BIM\pyWHA\pyWHA.extension"
==> Default Extension Search Path
C:\Users\mpfammatter\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
W:\01 BIM\pyWHA
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
2019.2.2 Hotfix | Version: 19.2.20.24 | Build: 20190808_0900(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2019"
2018 First Customer Ship | Version: 18.0.0.420 | Build: 20170223_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2018"
2017.2.5 Security Fix | Version: 17.0.1169.0 | Build: 20190508_0315(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2017"
2016 Service Pack 2 | Version: 16.0.490.0 | Build: 20150714_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2016"
2015 Update Release 3 | Version: 15.0.207.0 | Build: 20140606_1530(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2015"
==> Running Revit Instances
==> User Environment
Microsoft Windows 10 [Version 10.0.18362]
Executing User: WRIGHT\mpfammatter
Active User:
Admin Access: No
%APPDATA%: "C:\Users\mpfammatter\AppData\Roaming"
Latest Installed .Net Framework: 4.8
No .Net Target Packs are installed.
No .Ne-Core Target Packs are installed.
pyRevit CLI 0.25.0.0
```

**Comment by eirannejad** [comment]
Merged into https://github.com/eirannejad/pyRevit/issues/884
